fedora

```diff
@@ -11,7 +11,7 @@ RUN ( dnf -y update || dnf -y update ) && \
 RUN if [ "${INTEL}" = "yes" ]; then \
   printf "[oneAPI]\nname=Intel oneAPI\nbaseurl=https://yum.repos.intel.com/oneapi\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=1\ngpgkey=https://yum.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB" > /etc/yum.repos.d/intel-oneapi.repo && \
   dnf -y update && \
-  dnf -y install intel-oneapi-compiler-dpcpp-cpp intel-oneapi-compiler-fortran intel-oneapi-compiler-dpcpp-cpp-and-cpp-classic intel-oneapi-mkl-devel && \
```

Comment:
Isn't the fortran compiler used somewhere in Kokkos.

Reply:
Only in an example that needs to be compiled manually, i.e., there is no `CMake` option to enable building it.

Reply:
But the container should have a fortran compiler, right? We are installing `flang` about, so I would think we should install `ifx` as well.

Reply:
What else apart from `Kokkos` are the ci-containers used for?

Reply:
Cabana! But even in the kokkos CI we use the Fortran compiler, so I would leave it in there.

Reply:
I have added it back, let's see if Docker can handle the image size.